### ISSUE TITLE
Flexibile start and end dates

### ISSUE THREAD

**Issue by xamanu** [grote/osm2gtfs#14]
@grote worte:

> For the dates, we need a more flexible solution, because when used with Navitia, the difference between start and end date can never be more than a year and ideally, for this use-case, I like these dates to automatically move forward when I run the script again without needing to update them somewhere manually.

Great idea. Maybe we can make the script to understand syntax like `now` and `+1year`, etc...

**Comment by grote** [comment]
> Maybe we can make the script to understand syntax like now and +1year, etc..

That might be overkill. How about the following?

```json
"feed_info" : {
  ...
  "validity" : {
    "dates" : "fixed",
    "start_date" : "0000-00-00",
    "end_date" : "0000-00-00"
  }
}
```
and this:
```json
"feed_info" : {
  ...
  "validity" : {
    "dates" : "automatic"
  }
```
Automatic then just takes the current data minus one month as start date and one year later as end date.

*Or* we make it even simpler and use automatic mode automatically when start_date and end_date are not specified. That's probably better.

There should also be a warning if the feed is valid for more than a year.

**Comment by xamanu** [comment]
Good proposal. I think this is the way to go.

**Comment by xamanu** [comment]
Implemented in #45
